synapse/lib/modelrev.py

```diff
@@ -993,6 +997,94 @@ async def save():
             if nodeedits:
                 await save()
 
+    async def _normFormSubsByProp(self, layers, formname, propname):
+        # This is basically self._normFormSubs but instead of norming subs based on the primary property value, we are
+        # normalizing subs based on the value of a secondary property
+
+        meta = {'time': s_common.now(), 'user': self.core.auth.rootuser.iden}
+
+        subprops = {}
+
+        form = self.core.model.form(formname)
+
+        nodeedits = []
+        for layr in layers:
+
+            async def save():
+                await layr.storNodeEdits(nodeedits, meta)
+                nodeedits.clear()
+
+            genr = layr.liftByProp(form.name, propname)
+
+            async for _, buid, sode in genr:
+
+                sodevalu = sode.get('valu')
```

Comment:
This logic would preclude us normalizing a secondary property which was flying solo in the storage node. We should probably just let the prop check below stand.